packages/web-app-files/src/components/SideBar/TagsPanel.vue

```diff
@@ -73,8 +73,8 @@ export default defineComponent({
     const store = useStore()
     const { graphClient } = useGraphClient()
 
-    const displayedItem = inject<Resource>('displayedItem')
-    const resource = computed(() => unref(displayedItem))
+    const injectedResource = inject<Resource>('resource')
```

Comment:
Not needed. You can reduce this to one line:
```js
const resource = inject<Resource>('resource');
```

Reply:
Yeah I thought so too, but then the unit tests fail horribly with `Cannot read properties of undefined (reading 'isRunning')`. I don't know why this as any effect on the task, but it has something to do with the watcher we have on `resource`.